kratos/solving_strategies/builder_and_solvers/residualbased_elimination_builder_and_solver.h

```diff
@@ -16,17 +16,12 @@
 
 /* System includes */
 #include <set>
+#include <unordered_set>

```

Comment:
note that to get the behaviour you had before, you need to actually define USE_LOCKS_IN_ASSEMBLY

Reply:
As it is in the block_builder_and_solver:

https://github.com/KratosMultiphysics/Kratos/blob/503f7b8073816a65696819af7c5f4aa1ef450dea/kratos/solving_strategies/builder_and_solvers/residualbased_block_builder_and_solver.h#L27